integration-tests/hawkular-server-tests/src/test/groovy/org/hawkular/metrics/AuthenticationITest.groovy

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright 2014-2016 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hawkular.metrics
+
+import groovy.json.internal.Charsets
+import groovyx.net.http.ContentType
+import groovyx.net.http.RESTClient
+import org.junit.BeforeClass
+import org.junit.Test
+
+import static org.junit.Assert.assertEquals
+
+/**
+ * @author jsanda
+ */
+class AuthenticationITest {
+
+  static String testUser = 'jdoe'
+  static String testPasword = 'password'
+  static String baseURI
+  static Closure defaultFailureHandler
+
+  @BeforeClass
+  static void initClient() {
+    defaultFailureHandler = { resp ->
+      def msg = "Got error response: ${resp.statusLine}"
+      if (resp.entity != null && resp.entity.contentLength != 0) {
+        def baos = new ByteArrayOutputStream()
+        resp.entity.writeTo(baos)
+        def entity = new String(baos.toByteArray(), Charsets.UTF_8)
+        msg = """${msg}
+=== Response body
+${entity}
+===
+"""
+      }
+      System.err.println(msg)
+      return resp
+    }
+
+    String host = System.getProperty('hawkular.bind.address') ?: 'localhost'
+    if ("0.0.0.0".equals(host)) {
+      host = "localhost"
+    }
+    int portOffset = Integer.parseInt(System.getProperty('hawkular.port.offset') ?: '0')
+    int httpPort = portOffset + 8080
+    baseURI = "http://${host}:${httpPort}/hawkular/metrics/"
+  }
+
+  @Test
+  void noCredentialsShouldFail() {
+    RESTClient client = createClientWithoutCredentials()
+
+    def response = client.get(path: "status")
+
+    assertEquals(500, response.status)
+  }
```

Comment:
For consistency's sake, it should be a FORBIDDEN, and should have been returned from Accounts. If it's 500, and if it's being thrown by Accounts, then something is wrong.